### PR TITLE
Adding new Git docs

### PR DESCRIPTION
**Pull request by joscdk**
Adding documentation to new git system

### CODE REVIEW

File: docs/advanced/general/using-private-git.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
 Using a private GIT repository is quite simple. Depending on if you are using GitHub or another git provider.
 
-## GitHub
-
 There are 2 ways you can grant access to a private repository on Github
 
-### Invite our Cloud API user
+## Invite ouri GitHub Cloud API user
```
Comment (Joannis) [inline docs/advanced/general/using-private-git.md]: our*

File: docs/advanced/general/using-private-git.md [hunk 2]
```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
...
+
+**Subject:** Add SSH Key: **[Organization name]**
+
+Attach the SSH key as a file to the Email.
 
-You can use the same SSH Key as in 'Another git provider'. This can be setup on either your own user or a user you create yourself.
+If you don't have an SSH Key, you can see how to create a new one [here](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
 
-## Another git provider (Gitlab, Bitbucket or self hosted)
+The file is usually located at `~/.ssh/id_rsa`.
```
Comment (Joannis) [inline docs/advanced/general/using-private-git.md]: id_rsa is the *private* key. 
`~/.ssh/id_rsa.pub` is the public key

File: docs/advanced/general/using-private-git.md [hunk 3]
```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
...
+
+### Add your SSH Key.
+
+As this system is still in beta, you need to send your key to our support system, and they will manually add it to the system. Be aware there can be a slight delay in adding the key. You will get a reply once it's added.
+
+We will later create an automatic system to manage the keys.
+
+Send an Email to `support@vapor.cloud`
+
+**Subject:** Add SSH Key: **[Organization name]**
```
Comment (Joannis) [inline docs/advanced/general/using-private-git.md]: You must add the _public_ key, explicitly noting that is important

File: docs/advanced/general/using-private-git.md [hunk 4]
```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
 
 Link to profile: https://github.com/vapor-cloud-api
 
-### Use SSH Key
+## Add your SSH Key to Vapor Cloud
+
+!!! info
```
Comment (Joannis) [inline docs/advanced/general/using-private-git.md]: Is `!!!` a markdown feature?

File: docs/advanced/general/using-private-git.md (hunk 2 above)
Comment (joscdk) [reply docs/advanced/general/using-private-git.md]: We need the private key, to get access to the Git repo :) .. It's the one they need to send us

File: docs/advanced/general/using-private-git.md (hunk 4 above)
Comment (joscdk) [reply docs/advanced/general/using-private-git.md]: Yes, it adds a box in a color based on the text info, warning etc.

File: docs/advanced/general/using-private-git.md [hunk 5]
```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
...
+    This will not be online before September 17 23:45 UTC. You can already now send your key in, and it will be onlien when we roll the system out.
+
+!!! info
+    SSH Keys will be linked to the organization, so they are automatically available to all users and projects under your organization.
+
+### Add your SSH Key.
+
+As this system is still in beta, you need to send your key to our support system, and they will manually add it to the system. Be aware there can be a slight delay in adding the key. You will get a reply once it's added.
```
Comment (Joannis) [inline docs/advanced/general/using-private-git.md]: `Be aware, ` with comma.

File: docs/advanced/general/using-private-git.md [hunk 6]
```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
 
 Link to profile: https://github.com/vapor-cloud-api
 
-### Use SSH Key
+## Add your SSH Key to Vapor Cloud
+
+!!! info
+    This will not be online before September 17 23:45 UTC. You can already now send your key in, and it will be onlien when we roll the system out.
```
Comment (Joannis) [inline docs/advanced/general/using-private-git.md]: online*

**Pull request merged**